src/step.js

```diff
@@ -22,6 +23,34 @@ export function Step() {
   renderRocket()
   airfighter.render()
 
+  if(mobileVelocity.createdJoystick) {
```

Comment:
It sounds weird that velocity has `.createdJoystick`.
Maybe `mobileJoystick` would be a better name for this object?
And also `createdJoystick` could become `if (mobileJoystick.isShown)`.

Secondly, we could also have something like `if (isMobileDevice)` instead.
Maybe this way the logic would be easier to understand.

Reply:
- [x] finished